### ISSUE TITLE
Referring to an enum from prose

### ISSUE THREAD

**Issue by annevk** [WICG/background-fetch#118]
"Background fetch response" refers directly to an IDL enum from prose, expecting the reader to expand it. I'm not aware of any precedent for that and interweaving IDL with prose like that seems a little weird.

I'd prefer if we kept some separation between the model and the external API.

**Comment by jakearchibald** [comment]
Is the best thing to do here to repeat the values in each place? That's what I was trying to avoid.

**Comment by annevk** [comment]
Yeah, model/prose values and IDL values. I guess we could try to figure out a new model for this, but the bit of duplication doesn't hurt that much I think.
